nanofi/src/api/nanofi.cpp

```diff
@@ -544,7 +544,7 @@ processor *add_python_processor(flow *flow, processor_logic* logic) {
     return nullptr;
   }
   auto lambda = [logic](core::ProcessSession *ps, core::ProcessContext *cx) {
-    logic(static_cast<processor_session*>(ps), static_cast<processor_context*>(cx));  //Meh, sorry for this
+    logic(static_cast<processor_session*>(ps), static_cast<processor_context*>(cx));  // Meh, sorry for this
```

Comment:
Does http://whatthecommit.com take user submissions?  This comment would be a good candidate for that. :)